src/lstm/recodebeam.cpp

```diff
@@ -657,20 +667,46 @@ void RecodeBeamSearch::ComputeTopN(const float *outputs, int num_outputs, int to
       }
     }
   }
+  float top_key = 0.0F;
+  float second_key = 0.0F;
+  bool found_first_code = false;
+  bool found_second_code = false;
   while (!top_heap_.empty()) {
     TopPair entry;
     top_heap_.Pop(&entry);
+    if (in_possible_diplopia_ && entry.data() == first_diplopia_code_)
```

Comment:
Please use {}.

Reply:
Hi Stefan,

Thanks for reviewing my code.  As this is the first time I have created a pull request, I am still learning the ropes and have a couple of questions.

For the comments you have made, should I make those corrections and then recommit to my branch JDWDIPLOPIA?  And then do I use the 'resolve conversation' button to let you know?

Dave

Reply:
Yes, you can recommit to your branch, either by amending to existing commits (then you have to use a forced push) or by adding more commits. GitHub sometimes resolves conversations automatically.